cmd/run_controller/main.go

```diff
@@ -39,15 +38,17 @@ func main() {
 	// creates the in-cluster config
 	var config *rest.Config
 	var err error
+	defer klog.Flush()
+
 	if kubeconfig == "" {
-		log.Printf("In cluster")
+		klog.Infof("In cluster")
```

Comment:
I think level zero for the info messages in this file is not appropriate. Most of them deserve 2 or even higher from my point of view.